mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -244,15 +385,17 @@ class Completions(
         val values = FilenameCompletions.contribute(filename, td)
         (values, true)
       case (lit @ Literal(Constant(_: String))) :: _ =>
-        val completions = InterpolatorCompletions.contribute(
-          pos,
-          completionPos,
-          indexedContext,
-          lit,
-          path,
-          this,
-          config.isCompletionSnippetsEnabled(),
-        )
+        val completions = InterpolatorCompletions
+          .contribute(
+            pos,
+            completionPos,
+            indexedContext,
+            lit,
+            path,
+            this,
+            config.isCompletionSnippetsEnabled(),
+          )
```

Comment:
As far I can see there are no changes here, only formatting, can you revert it? (less modifications is better)